src/PageApi.php

```diff
@@ -139,7 +140,7 @@ public function call($method, $url, $withPath = true, $withCache = true)
 
         try {
             if (!$shouldCache || !$value = $this->cache->get($cacheKey)) {
-                $response = $this->http->request(strtoupper($method), $url);
+                $response = $this->http->request(strtoupper($method), $url, ['json' => $body]);
```

Comment:
This is fine on a GET request? Seems unnecessary to set `json` to `null` if there is no `$body` passed.

Reply:
It is fine on a `GET` request; I'm also checking for `$body` before setting it now 🫡